chapters/appendix-IV-SPDX-license-expressions.md

```diff
@@ -1,175 +1,174 @@
+### IV.3.2 Exception `WITH` Operator <a name="IV.3.2"></a>
 
-states the OR operator should be applied before the AND operator. That is, one should first select between the LGPL-2.1+ or the BSD-3-Clause license before applying the MIT license.
+A License Exception can be expressed in XML via a `<spdx:LicenseException>` element.  This element has the following children:
 
-### 5) License Expressions in RDF <a name="rdf-expr"></a>
+* [`rdfs:comment`][rdfs:comment] element describing the nature of the exception.
+* [`rdfs:seeAlso`][rdfs:seeAlso] element (optional) referencing external sources of information on the exception.
+* `spdx:example`: Text describing examples of this exception.
+* `spdx:name`: The full human readable name of the item.
+* `spdx:licenseExceptionId`: The identifier of an exception in the SPDX License List to which the exception applies.
 
-A conjunctive license can be expressed in RDF via a `<spdx:ConjunctiveLicenseSet>` element, with an spdx:member property for each element in the conjunctive license. Two or more members are required.
+    FIXME: I don't think this element makes sense.  What you want is a way to represent a `simple-expression` based on `WITH`.  This subsection seems to be focused on defining exceptions, but it does not cover *consuming* those license exceptions.  And “and exception … to which the exception applies” is really not clear; is it the left or right `WITH` operand?  What happens for licenses/exceptions that are not in the SPDX License List?
+* `spdx:licenseExceptionText`: Full text of the license exception.
+
+<!-- the following code block is not part of the preceding list -->
+
+    <rdf:Description rdf:about="http://example.org#SPDXRef-ButIdDontWantToException">
+        <rdfs:comment>This exception may be invalid in some jurisdictions.</rdfs:comment>
+        <rdfs:seeAlso>http://dilbert.com/strip/1997-01-15</rdfs:seeAlso>
+        <spdx:example>So this one time, I had a license exception…</spdx:example>
+        <spdx:licenseExceptionText>
+            A user of this software may decline to follow any subset of the terms of this license upon
+            finding any or all such terms unfavorable.
+        </spdx:licenseExceptionText>
+        <spdx:name>&quot;But I Don&apos;t Want To&quot; Exception</spdx:name>
+        <spdx:licenseExceptionId>SPDXRef-ButIdDontWantToException</spdx:licenseExceptionId>
+        <rdf:type rdf:resource="http://spdx.org/rdf/terms#LicenseException"/>
+    </rdf:Description>
+
+### IV.3.3 Conjunctive `AND` Operator <a name="IV.3.3"></a>
+
+A [conjunctive license](#IV.2.3) can be expressed in XML via a `<spdx:ConjunctiveLicenseSet>` element, with an `spdx:member` property for each element in the conjunctive license.  Two or more members are required.
 
     <spdx:ConjunctiveLicenseSet>
         <spdx:member rdf:resource="http://spdx.org/licenses/GPL-2.0"/>
         <spdx:ExtractedLicensingInfo rdf:about="http://example.org#LicenseRef-EternalSurrender">
```

Comment:
I think this is a missing <spdx:member> which should enclose the ExtractedLicensingInfo

Reply:
Addressed in cf8decf → 9c8d189.